.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - run: sudo apt update
+    - run: sudo apt install libpng-dev libldap2-dev php7.2-zip php7.2-bcmath php7.2-gd php7.2-ldap
+    - name: Update Composer
+      run: sudo composer self-update
+    - name: Cache vendor directory
+      uses: actions/cache@preview
+      with:
+        path: vendor
+        key: ${{ runner.os }}-composer
+    - name: Install Composer dependencies
+      run: composer install --no-interaction --no-progress --no-suggest --no-dev
```

Comment:
There are a couple more good flags to throw on here, look at the deployment script.

Also, you can pass `--ignore-platform-reqs` to get it to ignore extensions and some other things to save build time at L13. Since we don't actually execute much PHP code during build it should be fine.